content/docs/command-reference/import-url.md

```diff
@@ -146,7 +146,7 @@ $ git checkout 2-remote
 $ mkdir data
 ```
 
-You should now have a blank <abbr>workspace</abbr>, just before
+You should now have a blank <abbr>workspace</abbr>, before
```

Comment:
Not condescending here either. "Just before" is a specific phrase meaning something like "the very prior step".

Reply:
Agreed!